docs/src/content/docs/it/reference/configuration.mdx

```diff
@@ -452,3 +536,20 @@ starlight({
 ```
 
 Consulta il [Riferimento alle sostituzioni](/it/reference/overrides/) per i dettagli di tutti i componenti che puoi sovrascrivere.
+
+### `plugins`
+
+**tipo:** [`StarlightPlugin[]`](/it/reference/plugins/#riferimento-rapido-per-le-API)  
```

Comment:
Qua viene mostrato un errore dal link check però dovrebbe essere risolto in #1450.